tests/installation/install_and_reboot.pm

```diff
@@ -68,9 +64,13 @@ sub run {
         push(@tags, 'screenlock');
     }
     # SCC might mean we install everything from the slow internet
-    if (check_var('SCC_REGISTER', 'installation')) {
+    if (check_var('SCC_REGISTER', 'installation') && !get_var('SCC_URL')) {
         $timeout = 5500;
     }
+    # aarch64 can be particularily slow depending on the hardware
+    $timeout *= 2 if check_var('ARCH', 'aarch64') && get_var('MAX_JOB_TIME');
```

Comment:
isn't every job has MAX_JOB_TIME defined ? I mean get_var('MAX_JOB_TIME') should always return true ?

Reply:
maybe but that is not related to my change because I did not change that, just move it